packit/api.py

```diff
@@ -274,7 +273,11 @@ def sync_from_downstream(
             for raw_file in raw_sync_files
             if Path(raw_file.dest).name not in exclude_files
         ]
-        sync_files(reverse_raw_sync_files, fail_on_missing=False)
+        sync_files(
+            self.dg.local_project.working_dir,
+            reverse_raw_sync_files,
+            fail_on_missing=False,
+        )
 
         if not no_pr:
```

Comment:
...somewhere here

Reply:
@TomasTomecek I don't understand why to function `sync_from_downstream`? I guess README.packit should be copied only from upstream to downstream. Do we really want to copy the file also to upstream? There is already `.packit.yaml` file which indicates that upstream repository is used by packit. I guess this could be only in `sync_release` function.

Reply:
oh, good point, wrong line